project-auth.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: Project Auth
+description: "Manage auth for your project"
+---
+
+By default, Hypermode protects your project's endpoint with an API key, but you
+can also bring your own auth too.
+
+## Retrieve API key
+
+To use the Hypermode API key, navigate to **Settings** → **API Keys** in your
+project dashboard. From there, you can copy your API key to a secure location
+for your app to consume.
+
+## Rotate API key
+
+There might be times when you need to rotate your API key. This could be due to
+a security breach, accidentally leaking your API key, and so on.
+
+Before rotating your API key, navigate to **Settings** → **Auth** in your
+project dashboard. Take note of the name of the `PEM key` with `hypermode-` as
+the prefix. You will need to delete this as the final step of this process.
```

Comment:
Yeah - I understand why they need to do this, but unless they read these docs very carefully, it's likely they won't.  Then they'll get to the final step and not know which to delete.

Reply:
should we expose when the API key was created so they can easily find the oldest one?